### ISSUE TITLE
Dailt status - May 2021

### ISSUE THREAD

**Issue by seanpm2001** [seanpm2001/seanpm2001#9]
***

### Daily status - May 2021

This is the daily status thread for May 2021. Daily status posts will be put daily (or when I get to it) below this post.

[Click here to see last months daily status posts](https://github.com/seanpm2001/seanpm2001/issues/7)

***

**Comment by seanpm2001** [comment]
***

### Status update: May 1st 2021

I had a very good day with development today. Today is the first day of May 2021, and I have a clean slate of progress today. Not all the data from May 1st comes in until the next day

I completed my journaling and gaming sessions early in the day again today. I tried a solution to my Saturday addiction problem. It really helped, but I don't think the source motivation was the 2 hour limit, I feel more like it is the battery limit. Luckily, I couldn't get the second game to load, so I ended up saving lots of time. My laptop continued to have battery problems today, and it crashed once. I might update my technical support article for mouse problems at some point, but I feel like I need more data first.

I have my first possible copycat on GitHub, although it might just be the person not knowing about the existing project. Someone made another Minecraft Python Edition project, with the same codename, but no source code. I have no power here, and I don't want to come across as evil, and anything I do to hurt the project would be bad. Imitation is the sincerest form of flattery they say, although this might not be imitation, due to how many attempts there have been to make an open source version of Minecraft. Their version doesn't give off much info, but I will promote it [here anyways, just in case it makes progress](https://github.com/RedisKritov/mcpye)

I have a new GitHub organization project I have been working on today pretty heavily, I plan to create 7 organizations (1 row) per day, as I have a ton of ideas and topics that need a better platform. I have mostly been using GitHub organizations as projects or topics. I plan to exceed the 10 organization limit this month, but I won't be making more than 7 per day.

I got some progress made today. I decided to quit and go to bed a bit early, as I ran out of things I wanted to do, and I want to work on my schedule.

Today was a very good day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: May 2nd 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `23` _(as of May 2nd 2021 at 11:59:59 pm)_

I had a very difficult day today mentally. A series of touchpad calibration issues led to a ton of mental, emotional, and stress problems for me today.

I have not had a good day today, and due to constant crashes, and other issues, my work got constantly delayed and my schedule was screwed up. I am under constant stress of machine failure, and it is not getting easier over the years.

Today, I continued the GitHub organization project, but didn't do anything unique today besides different documentation. I have started to write a year in review for my first year of GitHub, which I have 23 more days to finish writing (goal) but can go longer if needed.

A family member has noted that the project that I mentioned yesterday was not a copycat, it was plagiarism. I still don't see it as plagiarism, as it isn't the official name, and as I have said, there have been many attempts to recreate Minecraft.

Today was not a very good day for development.

***

**Comment by seanpm2001** [comment]
***

### Status update: May 3rd 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `22` _(as of May 3rd 2021 at 11:59:59 pm)_

I had a good day with development today. I started a small change in my day (redacted) and it made it a lot easier for me today. Despite getting less than 8 hours of sleep last night, I did much better than I normally did, all the way to the end, with no errors.

Today, I finally ratified a decision on the release of a journaling module I considered sensitive. I decided it was fine to put out, and I committed a stripped down version that complies with GitHubs terms of service and file size limit. My sexuality center is now active. It took a long decision to do this.

Today, I also worked on some new projects that I came up with this morning and during my dental appointment, which was a new game based around slavery, and why it should be completely abolished (because unfortunately, slavery still exists in this world, and some of the biggest slave holders include Nestle, and certain other chocolate producers) I want to make this game with limited restrictions to show people how bad slavery is and to help push more people into doing something about freeing the slaves in the 21st century.

Another important project I started today was an article on laptops and desktop computers that come pre-installed with Linux, and info about them. I hope for it to be a researched catalogue (I am keeping the ue on this word, ignoring US English standards. I like it better this way) of Linux devices to make it easier for people to switch to Linux, and find a good Linux machine. I require experiences in this, and I have only had 2 Linux laptops, so I didn't have very much to contribute.

I had 2 medical appointments today, 1 mental, and 1 physical. My dentist appointment went great, and during a little chat with one of the dentists, I had an increased interest in Botony (Plant biology) and want to pick up another part-time hobby. One goal I have in mind is to increase the production of raspberries, so that they don't cost $6.00+ for a small package in stores, and so they are more easily accessible. They are my favorite fruit, but they are very difficult to harvest. That is why I am planning a machine that can do so, if one doesn't exist already. Otherwise, I don't see why they are so expensive.

Some other project ideas that came to mind today were for more image projects, and a proper journaling package for my opinion set.



Other than all this, I did the usual GitHub work today, and also managed to do the organization work and my journaling work. I didn't have any time to work on my year in review article.

I am really tired, and I missed some sleep last night, so I will be going to bed soon/

Today was a very good day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 4th 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `21` _(as of May 4th 2021 at 11:59:59 pm)_

I had a good day with development today. I had my monthly gaming session and successfully managed to quell the addiction and save myself over 4 hours of time.

In Geek culture today, it was Star Wars day today (**May** the **4th** be with you) I have never been much of a Star Wars fan (to be honest, I have never seen any of the Star Wars movies in full, I have only seen about 8 minutes of footage across all films) I did include 1 May 4th reference in a commit today.

I got some work done today, and I was in a generally good mood. I didn't make any progress towards my year in review article today, but I did work on Tetris 128 a bit (not very many commits were made, but a decent amount of time was spent compared to other projects)

I did further GitHub organization work today, adding 7 new projects. I generated more ideas, so this is going to drag on further unless/until a limitation is reached. I had to quit as it got close to midnight.

Today was a good day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 5th 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `20` _(as of May 5th 2021 at 00:12:00 am to 11:59:59 pm)_

I had an OK day with development today. I had so much available time to work today, but I didn't get as much done as I thought I would, but I still did a lot.

Today marks the introduction of a new feature that will be added to all my workflows, which is the dropdown feature. All markdown documents I make now that have a file history section will contain a dropdown, it is incredibly easy to make, I wish I would have learned this sooner.

I did work in various projects today, and listened to music on my phone while working. It is a really good system, and I really like it. It has brought me new experiences.

I stayed up pretty late, I hoped to go to bed earlier, but I didn't.

Today, I sent my previous laptop to a repair store in Oregon, @priceComputersLLC my family has a brief history with them, and they seem incredibly trustworthy. Going into the store gave me really good unexplainable vibes. I am hoping the issue can be fixed, but at the same time, it may be difficult to transition back to computer 1 again. Maybe a few system errors will sway my view.

Today, I updated my main profile README.md file twice, one update to add new content, and update to finalize a successful dropdown test.

I created 7 more organizations today. I am currently creating a 7x3 staircase of SNU projects in my GItHub organization listing.

I did some work on the Tetris128 project today, making it further with the concept, and laying out a roadmap. I plan on having as much Linux support as possible to boost Linux desktop share even further, and decreasing Windows and MacOS share slightly (this whole plan will only work if the game gains some traction)

I am deciding on how to how to make the game run with as little CPU as possible. The main problem being that playing with softbody graphics either requires a killer $10000 gaming computer, or a gigantic triage of efforts made to make the engine lighter. The demo I have contains less than 2 minutes of footage, and is reported to have taken ~36 hours to render. My plan is to be able to do this, but in real time (36 hours -> 60-1000 times per second) it is a very difficult goal, but I feel this, along with the other features (such as party mix, playing with other polyominoes, and the customization and open-ness of the game will completely revolutionize Tetris. I might even add in a battle royale mode if it is really needed.

Today, I very briefly worked on Mammal Town, laying the foundation to start adding characters to the game. Animal Crossing New Leaf was my inspiration for getting back into this. I only added 1 character today, so I didn't do much.

Today, I attempted some more torrents, but torrent 2 failed, as all the web seeds left and expired, leaving me stranded with a time estimation going towards the end of the Universe. So I just canceled it and did a direct download for the current and next 2 would-be torrents.

I worked briefly on the SNU 2D Translation service today, adding new language support. In the process of research, I find it weird how the system font recognizes Ancient Phoenician characters (~12000 BCE, about 14050+ years ago) yet it can't fully decode Javanese, or Mongolian, and some other languages I don't remember right now but use occasionally.

I worked on my text messaging instruction repository today, there was a lot of cleanup, fixes, and rewrites to do. It honestly looked like I was incredibly drunk while making it originally, yet I can remember that I actually thought it looked good back then. It is amazing how much I can change in just a few months.

Due to all these new topics/projects/Organizations, I have been doing a lot more forks. I have had a forking frenzy today.

Today, I played around a little bit on Gist.GitHub.com, as it is where I discovered using dropdowns in Markdown (which uses HTML5) I also learned something in HTML5, which is the `open` keyword will make the details of a dropdown display automatically. I don't currently see very much use in this for myself, but it might come in handy in the future.

```html
<details open>
<summary>This is open</summary>
<p lang="en">(👁️👄👁️) YOU WEREN'T SUPPOSED TO OPEN THE BOX DAVID</p>
<p lang="en">| | | - | | | _</p> <!-- Is this loss? !-->
</details>
```

I very briefly worked on my anti-slavery game today, just adding in some resource notes, and keeping the project active.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today.

Today was a decent day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 6th 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `19` _(as of May 6th 2021 at 00:12:00 am to 11:59:59 pm)_

I had an OK day with development today. 
Today marks the day since the introduction of a new feature that will be added to all my workflows, which is the dropdown feature. All markdown documents I make now that have a file history section will contain a dropdown, it is incredibly easy to make, I wish I would have learned this sooner. I didn't implement this anywhere today.

I did work in various projects today, and listened to music on my phone while working. It is a really good system, and I really like it. It has brought me new experiences. I have just been letting the playlist play instead of finding songs. 

I stayed up pretty late, I hoped to go to bed earlier, but I didn't.

I created 7 more organizations today. I am currently creating a 7x3 staircase of SNU projects in my GItHub organization listing. This goal failed today, and the 1st and 2nd stairs are now wobbly objects.

Today, I worked on Mammal Town, laying the foundation to start adding characters to the game. Animal Crossing New Leaf was my inspiration for getting back into this. I added 4 characters today, and heavily increased documentation.

Due to all these new topics/projects/Organizations, I have been doing a lot more forks. I have had another forking frenzy today.

Sorry for the terrifying GitHub dropdown yesterday featuring the details HTML tag, looking back, I really should not be writing documentation late at night this carelessly, and I should refrain from anything that tries so hard to be funny when it is being done at night. It is now on the growing pile of past embarrassments that I can't get out of my head (hopefully what I am writing now doesn't get added to that pile)

I did not do any work to my anti-slavery game today. I didn't know what to do with the project.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today.

Today was a decent day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 7th 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `18` _(as of May 7th 2021 at 00:12:00 am to 11:59:59 pm)_

I had a decent day with development today. 


I stayed up pretty late, I was in a rush with the last hour of my day after procrastinationg a lot.

I created 7 more organizations today. I hoped to get closer to finishing my organization goal, but I am now expecting it to drag on to May 21st 2021 or further.

Today, I mostly did documentation-related work. I didn't do any full-on programming today. My laptop has been incredibly unstable lately, and this has been an issue for 2 different laptops now (one Windows and one Ubuntu) and I am working on narrowing the issue down and getting it down to a science, but it is difficult, as it is related to bad drivers, and there are over 80 different factors of what it could be. Here is my current listing:

```ruby

# Crash data

# Ruled out

###

Certain point of screen

###

# Still possible problems

###

Printer conflict
Firefox 88
Gedit
LibreOffice
VLC Media Player
Heavy saving and file io
Large file opening
GNOME extensions/desktop
Desktop icons
Desktop conflict
Xorg
Too much RAM
Too much SWAP
Low disk free space
GNOME Tweaks
Screen animations
Battery related problems
Driver related problems
Ubuntu system error problems
Video graphics issue
Modprobe psmouse -r
A bad set of updates
Extensions (GNOME)
Samba conflict
SVG file conflict
Image viewer conflict
Battery conflict
VirtualBox driver conflict
A1 minute mouse freeze
B2 minute mouse freeze
Touchpad callibration
Uncommon websites
Bad system activity
Application general issue
Super key
Key switch issue (SHIFT stuck)
Alt key stuck
CTRL key stuck
FN key stuck
FN key conflict
Scrolling error
Khan Academy
GitHub conflict
Wi-Fi conflict
Bluetooth conflict
Modprobe psmouse
System error
Microsoft Word do cument for mat
SD card conflict
Charger conflict
Bad block of memory being touched
GNOME files
Too many files on the desktop
Too many image files
High CPU usage problems
Files with 99 or more lines of code
Mass GitHub works
Bad network seeds
Network port attack
Application conflict/other
Clock conflict
Battery percentage conflict
Wi-Fi reader conflict
General video playback issue
General audio playback issue
Location permissions conflict
Issue with tracker miner fs
RAM underreported, overloading system
CPU underreported, overloading system
Saving webpage issue
Bad file character issue #(#, $, , | { etc. )
Transmission BitTorrent client conflict
Other Ubuntu problems/unknown
Other problems/unknown

###

```

I chose Ruby, as it highlights each entry with yellow text for the first word when capitalized (as of Gedit) 

Due to all these new topics/projects/Organizations, I have been doing a lot more forks. I have had another shorter forking frenzy today. Today I worked on the organization description repository and standardized it. Recently, GitHub added support for the Git `fetch-upstream` command, which will be very useful in the future for all these organizations of mine and their projects.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

Today was a decent day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 8th 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `17` _(as of May 8th 2021 at 00:12:00 am to 11:59:59 pm)_

I had a decent day with development today. 

I tried to do tasks other than image and documentation work on the usual projects today, I worked on some other lesser-worked on projects as well. This GitHub Organization task isn't eating too far into my time, I only work on it for 25-80 minutes per day. I went ahead and calculated the rest of the possible projects, and it is expected now to finish up on June 2nd 2021. More may be added, and it could get extended up to June 20th 2021, but I went through all my projects today to make sure.

Regarding screenshots, I am really considering what has to be done to change the project language on my GitHub image project to make it Ruby 50.1% instead of Go 100% as I don't want an extremely common project to be associated with this. Also, I plan on dropping the xx255 screenshot from the cycle, as it is too tedious to get both 255 and 256, and a bit unnecessary.

I created 7 more organizations today. I hoped to get closer to finishing my organization goal, but I am now expecting it to drag on to June 2nd 2021 or further (see above)

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

Today was a decent day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 9th 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `16` _(as of May 9th 2021 at 00:12:00 am to 11:59:59 pm)_

[See my year in review for Y1 (2020-2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021)

I had a decent day with development today. 

Today is Mothers Day (2021) I didn't make any Mothers Day type projects, but I have a new plan I am working on regarding user interface and hardware friendliness, as the elderly and people with disabilities seem to be widely ignored when it comes to this. A new problem I have found today is that older people have difficulty with smaller keyboards, and since most laptops are getting ridiculously small and less powerful (now most laptops require a dongle unfortunately, which I really hate) it is getting hard for my mom to get a new computer. We considered the Microsoft Surface Pro 7, but it appears the keyboard is just too small and it likely requires a dongle, which is a double no for her (for me, it would be a triple no, because Windows 10 is on it and it likely doesn't have good Linux support)

Another problem I have already been attacking is the presence of flat design, which makes user interfaces not only a lot uglier, but also over 16% more difficult to learn and use, not only for the elderly, but for everyone. Skeuomorphism really needs to come back, and I await the day it does. I have been suffering with garbage flat systems for nearly a decade now (last = iOS 6, Windows 7, Android 2.3 Gingerbread)

I did some actual programming today... well technically just shell scripting. I worked on my integer overflow by date project (YKDefender) and contributed 2 new original scripts (4 total) I had to re-learn MS-DOS Batch files slightly. I did some other work today as well, including higher level documentation.

I updated my GitHub profile README.md file today. Earlier this month, I discovered dropdowns in markdown, and learned a new way of doing it: `<details open>` I didn't think I would ever use it, but I found some useful uses for it today.

I created 7 more organizations today. I hoped to get closer to finishing my organization goal, but I am now expecting it to drag on to June 4th 2021 or further. I keep getting more new ideas. I am hoping to eventually finish this task.

Today, I surpassed 5000 stars on GitHub. In the next 2 days (by May 11th 2021) I will know if GitHub supports having 5000+ stars (as I know YouTube and Twitter don't, at least when I used to use them)

I renovated some old projects today and stayed up a bit late while working.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

Today was a decent day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 10th 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `15` _(as of May 10th 2021 at 00:12:00 am to 11:59:59 pm)_

[See my year in review for Y1 (2020-2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021)

I had a decent day with development today. 

Today for my mass-organization creation and registration process, I bumped up the speed from 7 organizations per day to 10. I won't go above this, as I want all of them to be able to show in each day of the calendar in full (if I were to create 11, it would cut off 1) this change has let me take the time down by 7 days, with a finish line back in May 2021 again. They aren't being made in rows, but this is necessary to complete this project faster and move on. It took a lot of time to do so today, and I made nearly 400 forks today in the process. It should be easier tomorrow.

In my image work today, I found that I didn't write day-by-day template status posts last Monday for my gaming sessions. I have corrected this today, although I don't know how I forgot in the first place.

Today I have found that one of my projects ([Its-time-to-cut-WideVine-DRM](https://github.com/seanpm2001/Its-time-to-cut-WideVine-DRM)) has reached 10 stars, my first project to reach double digits. I wish I could improve it more, but I don't have enough ideas at the moment. It is a big milestone for me. I have been watching this project for a few months, and I have seen it grow popular. Thanks to everyone who supported it. I will do what I can to continue to improve it. (I did make a small set of improvements today not mentioning this milestone, but I don't feel like it was enough)

I did some actual programming today... well technically just shell scripting. I worked on my integer overflow by date project (YKDefender) and contributed 2 new original scripts (4 total) I had to re-learn MS-DOS Batch files slightly. I did some other work today as well, including higher level documentation.

I created 10 more organizations today. I hoped to get closer to finishing my organization goal. I keep getting more new ideas. I am hoping to eventually finish this task. See above for more info

Today, I surpassed 5000 stars on GitHub. I can now confirm (at least for myself) that GitHub doesn't limit its users to 5000 stars/follows/likes like YouTube and Twitter do. I exceeded 5.3 thousand stars today.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

Today was a decent day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 11th 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `14` _(as of May 11th 2021 at 00:12:00 am to 11:59:59 pm)_

[See my year in review for Year 1 (2020-2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021)

I had a weak day with development today. 

I continued my goal of creating GitHub organizations for platform and registration, creating 10 more today. I woke up early today and was able to get work on this project done, although I decided to go to bed earlier tonight, as I want to fix my schedule.

I did lots of documentation work and optimizations to existing documentation today. Some notable documentation work was an update to my article against Google WideVine (DRM) my GitHub profile `README.md` file, and finalized the ProjectWiki in 3 projects. I am going to start getting paid soon, so I have to do some planning on how to sort my earnings (I have decided to invest half into spendable money that goes towards the essentials, like servers, service repair, personal expenses, and fuel) and half goes to stock up to start my career in computer business (equipment manufacturing, worker payment (a good wage, at least quadruple the current minimum wage for all workers, everyone will get paid the same) non-malicious advertising, among other things.

I have been doing a lot of archiving, sorting, and planning today, and also getting distracted with other projects. I didn't do very well today development wise.

I created 10 more organizations today. I hoped to get closer to finishing my organization goal. I keep getting more new ideas. I am hoping to eventually finish this task. See above for more info

Today, I surpassed 5500 stars on GitHub. I can now confirm (at least for myself) that GitHub doesn't limit its users to 5000 stars/follows/likes like YouTube and Twitter do. I exceeded 5.6 thousand stars today.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

Today was a decent day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 12th 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `13` _(as of May 12th 2021 at 00:12:00 am to 11:59:59 pm)_

[See my year in review for Year 1 (2020-2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021)

I had a very good day with development today, despite what I tell myself (lately, I haven't been able to determine what is considered "enough" work for a day)

I continued my goal of creating GitHub organizations for platform and registration, creating 10 more today. I woke up early today and was able to get work on this project done, although I decided to go to bed early again tonight, as I want to continue to fix my schedule

Today was a pretty big day today. I had a list of ideas to get through, and I am planning on resuming work on some older plans pretty soon. I am slowly working towards 1000 repositories, with a scope of everything I need.

Today, I had an influx of followers (4 followers in 1 day) as of 8:17 pm, I am up to 150 followers. I am also finally getting some more interaction on my projects. Today, I had my first external contribution my anti-Google WideVine article, somebody had the time to help replace a graphic with a transparent version. This is the 3rd time somebody other than me has contributed to one of my projects in my GitHub history, and I am grateful for every contribution. Thanks to the user [@GaurPeeyush](https://github.com/GaurPeeyush) for helping out today. (also, India is really affected by COVID-19 at the moment, and this user reports themselves to be from there. I hope you make it through this with minimal suffering, these are tough times. Good luck with your web development. I hope you are OK)

I have had my computer in for repair for a while now, I finally got to pick it up today. I am not yet fully ready to give a full review on Price Computers LLC, as I haven't made sure the data is safe. Once that is done, I am willing to give a 100/100 review. It is a really nice little computer store, and it gives off good vibes. They have also been in business as a small company for 39 years, so they had to have done something right to do so. So far, I have been able to tell that the operating system is reinstalled, which is great. The service there is really nice and professional, and they really seem to know what they are doing.

I didn't do very much file sorting today, I am still on my backup laptop and I am preparing to move back to the main one, once I recover my data onto it. It is still going to take up to a week or longer to transition back.

I created 10 more organizations today. I hoped to get closer to finishing my organization goal. I keep getting more new ideas, but the flow has slowed down, so that I am not making negative-positive progress again. I am hoping to eventually finish this task. See above for more info

Today, I surpassed 5700 stars on GitHub. I can now confirm (at least for myself) that GitHub doesn't limit its users to 5000 stars/follows/likes like YouTube and Twitter do. I exceeded 5.6 thousand stars today.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

Today was a decent day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 13th 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `12` _(as of May 13th 2021 at 00:12:00 am to 11:59:59 pm)_

[See my year in review for Year 1 (2020-2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021)

I had a very bad day today, but an OK day with development. Many things went wrong today, and it was generally very difficult for me. I gained at leaat 6 new problems today, of which 3 are major/critical.

I continued my goal of creating GitHub organizations for platform and registration, creating 10 more today. I woke up early today and was able to get work on this project done, although I decided to go to bed early again tonight, but it is starting to wane back towards going to bed later and later. I got less than 8 hours of sleep last night, and I have been getting woken up before 7:00 am, so this isn't good. I generated ideas for 7+ days worth of new organizations, and the estimate has been moved up to June 4th/June 5th 2021.

I tested out the laptop backup today. I can't comment too much on it yet, I need to message Price Computers tomorrow when available. I really wish I didn't take the advice of "its OK to not backup your data tonight, they wouldn't have their job if they didn't know what they were doing" as it has led to a series of unfortunate events that just keep getting worse. The new issue I am trying to fix now is getting all my files relocated. There were over 2 million files, and my estimate for getting everything fixed again is 609 hours of renaming, 3 hours per day (~40 million seconds) however, this calculation was probably not too good. The concern I am currently having is that the files aren't in their original folders, and don't have their original names, but a software tool was used to sort them by their file type. I might have to revert to an older backup for most data (luckily, I don't have to rename too many files, just ~100000+ files in the past 1.1-3.6 months, so my past calculation was pointless. I am just writing this description as I go, figuring it out in my head as I write more) I am hoping the metadata retained some filename info. At least the files aren't lost, that is the best part. I am pretty sure that this was done due to difficulty accessing the data. They even took the time to create 2 backup formats of system and app data along with the bootloader (GRUB) in FAT32 and HFS+ formats (no program data was lost either, that's a bonus) I am still not 100% on this, it caused a pretty big issue tonight, and I will need to look into it further.

I was a bit scared at first, as the drive wasn't recognized on a Windows 10 laptop, which I tried first, as I didn't want to use a dongle. Apparently, it was formatted directly for Linux only, not Windows. The laptop has a weird screen flickering problem now at bootup, but is fine afterwards. I also got a new hard drive out of this, which I dubbed AD-A (codename Ada Price) which will be my 2 terabyte drive for backups only (see more on my [personal server info repository](https://github.com/seanpm2001/Personal-Server-Info))



I created 10 more organizations today. I hoped to get closer to finishing my organization goal. I keep getting new ideas, and the flow went up by 7 days today, so I am making negative-positive progress again. I am hoping to eventually finish this task. See above for more info.

Today, I surpassed 5800 stars on GitHub. I am also rapidly gaining followers due to "investing" in some trends, such as GitHub education and Apache FreeMarker. I am just one of the few people who both watches and stars repositories.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

Today was an OK day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 14th 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `11` _(as of May 14th 2021 at 00:12:00 am to 11:59:59 pm)_

[See my year in review for Year 1 (2020-2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021)

I had a much better day today, but just an OK day with development.

I am able to finalize my review on the drive. I was on call with the store today, and I found where the files were located. I was upset over nothing last night, everything has worked out fine. My final review on Price Computers LLC is 100/100 for computer repair.



I created 10 more organizations today. I hoped to get closer to finishing my organization goal. I keep getting new ideas, and the flow went up by 7 days today, so I am making negative-positive progress again. I am hoping to eventually finish this task. See above for more info.

Today, I surpassed 6000 stars on GitHub. I didn't do much today, just more of what I did yesterday. I had more downtime today, and did mass forking. I have resolved all but 1 issue that came up yesterday, which is a high level of tension (TMJ) I am dealing with.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

Today was an OK day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 15th 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `10` _(as of May 15th 2021 at 00:12:00 am to 11:59:59 pm)_

[See my year in review for Year 1 (2020-2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021)

I had a better day today, and a good day with development.

I am able to finalize my review on the drive. I was on call with the store today, and I found where the files were located. I was upset over nothing last night, everything has worked out fine. My final review on Price Computers LLC is 100/100 for computer repair.



I created 10 more organizations today. I hoped to get closer to finishing my organization goal. I keep getting new ideas, and the flow didn't go up today, so I made positive-positive progress and not positive-negative progress (positive-negative means I made progress, but also didn't get much closer to the goal, as part of my progress was in moving the goal post forward)

Last night when browsing GitHub topics, I discovered a new purpose for my image projects. They won't just be for stock images and progress monitoring anymore, I have decided to expand their purpose by adding in machine learning support, as I have a pretty good collection compared to the popular one I found (unsplash) although my collection mostly leans on gaming, which would be important for machine learning. I am still trying to figure out how to configure this and display this across all the projects.

I have recently gained a much higher interest in Firefox profiles again recently, and I have been getting creative with it again. I spent some time doing so today.

Today, I worked on various project, and started work on the 6 new projects that I have had planned out in my notes for a bit.

Today, I surpassed 6100 stars on GitHub. My computer completely crashed earlier (completely unresponsive, had to use the 10 second hard power cycle trick) at `17:30:48` and I lost all of my network data up to this point (the system monitors collection of uploads/downloads rate, not a big loss. Download was near 500 MiB last time I checked, and upload was near 1.0 GiB. This was 3 minutes before the crash) I am wondering if accidentally summoning VLC Media Player last night caused this. I also noticed poor performance up to this point, as my system was lagging heavily. I am also suspecting high cache and bad memory and processor reports may have contributed to the crash (this laptop has been under-reporting how much CPU and RAM is being used, and if it reached max capacity, it would crash. For example, my laptop constantly reports 10-20+ percent CPU usage across all 8 cores combined, but when I look at the processes and daemons, all of them together don't even equal 5%) there were 2 additional crashes today, 1 wasn't fatal, 1 wasn't as fatal but still required a reboot (the additional 2 were cursor/desktop crashes) I plan to try and get my main laptop back up and running tomorrow.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

Today was a good day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 16th 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `09` _(as of May 16th 2021 at 00:12:00 am to 11:59:59 pm)_

[See my year in review for Year 1 (2020-2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021)

I had a better day today, and a decent day with development.

I created 10 more organizations today. I hoped to get closer to finishing my organization goal. I keep getting new ideas, and the flow didn't go up today, so I made positive-positive progress and not positive-negative progress (positive-negative means I made progress, but also didn't get much closer to the goal, as part of my progress was in moving the goal post forward) it still took over 3 hours today, and it set me back a bit, but I enjoyed doing it. I also did Firefox profile customization in the process (as I create a new profile for each organization) I have recently gained a much higher interest in Firefox profiles again recently, and I have been getting creative with it again. I spent some time doing so today.

Today, I worked on various project, and continued work on the 6 new projects that I have had planned out in my notes for a bit. I also finally made proper documentation for my `IGNORE.md` file system.

I really wanted to continue work tonight, but I was too unfocused. I hope to try to go to bed earlier so I have more time tomorrow. I have 2 medical appointments tomorrow morning.

Today, I surpassed 6300 stars on GitHub. I didn't do much on GitHub today other than the usual work.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

Today was a decent day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 17th 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `08` _(as of May 17th 2021 at 00:12:00 am to 11:59:59 pm)_

[See my year in review for Year 1 (2020-2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021)

I had a difficult day today, and a decent day with development.

I got woken up very early this morning and was unable to get back to sleep, meaning I got less than 5 hours of sleep last night. I tried everything I could last night to make this not happen, but it all failed.

I was in a lot of pain for most of the morning, but I was able to have a good time and be productive. I did my first external hard drive backup in over a month, and got my leisure gaming sessions completed early. I only had 1 medical appointment today, the next one is actually tomorrow. There was a lot of internal confusion towards this.

I created 10 more organizations today. I hoped to get closer to finishing my organization goal. I keep getting new ideas, and the flow didn't go up today, so I made positive-positive progress and not positive-negative progress (positive-negative means I made progress, but also didn't get much closer to the goal, as part of my progress was in moving the goal post forward) it took significantly less time today compared to yesterday, but there wasn't as much to do. 

Today, I worked on various projects, and continued work on 5 out of the 6 new projects that I have had planned out in my notes for a bit. I was going to do this last night, but I became too tired. I have to go to bed earlier tonight, so I am wrapping up early.

I really wanted to continue work tonight, but I needed to go to bed earlier to get better sleep tonight.

Today, I surpassed 6400 stars on GitHub. I didn't do much on GitHub today other than the usual work. I did resume work on 5 of the 6 projects I have been holding off. I am getting close to reaching 2 goals by May 25th 2021 (163 followers, 65536 commits) I feel like I can make the 2nd one, but the first one I am still unsure on.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

Today was a decent day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 18th 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `07` _(as of May 18th 2021 at 00:12:00 am to 11:59:59 pm)_

[See my year in review for Year 1 (2020-2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021)

I had a very good day with development today, and a pretty good day today.

I had a longer than expected gaming session this morning that didn't drag on for too long, and also got cut in half due to a doctors appointment over the phone that lasted less than 9 minutes but was an effective check-up.

I created 10 more organizations today. I hoped to get closer to finishing my organization goal. I keep getting new ideas, and the flow didn't go up today, so I made positive-positive progress and not positive-negative progress (positive-negative means I made progress, but also didn't get much closer to the goal, as part of my progress was in moving the goal post forward) it took less time today compared to yesterday again, but there wasn't as much to do. 

Yesterday, I doomed myself with even more organization work, but the work is necessary. I am now documenting all the individual organizations. This would have worked better if I had planned for it sooner.

Today, I worked on various projects, and continued work on 5 out of the 6 new projects that I have had planned out in my notes for a bit. I was going to do this last night, but I became too tired. I have to go to bed earlier tonight, so I am wrapping up early. I made lots of progress on the Google Drive module and Website Archives module, but felt that these projects were not as important as others.

Today, I did research on the Python programming language, and networking, so I can better myself. I learned a lot today via Wikiversity. I got to learn 6 new terms, and start remastering the first 11 chapters of my 2 years of Python class in High School.
Stayed up late again, even later

I got enough done today, and didn't need to stay up extra late. I still stayed up pretty late though. I also reached 2^16 (65,536 commits) today, which was a major accomplishment. I began finalizing work near 11:00 pm, and hope to make more progress tomorrow.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

Today was a very good day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 19th 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `06` _(as of May 19th 2021 at 00:12:00 am to 11:59:59 pm)_

[See my year in review for Year 1 (2020-2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021)

I had a very good day with development today, and a pretty good day today.

I had a longer than expected gaming session this morning that didn't drag on for too long. I focused heavily on documentation work today.

I created 10 more organizations today. I hoped to get closer to finishing my organization goal. I had several new ideas, and now it is at its max estimate of June 20th 2021. I have finally gotten to consistently enjoying doing this, so it should be fine. With the GitHub organization info repository, I made significant progress today and wrote 30 status entries, along with other documentation improvements. I am 25% caught up now (53 out of 212 organizations documented) and I plan to keep doing this consistently until I am caught up. I will have to continue at a faster pace of status files to catch up in 6-9-12 days.

Today, I did more research on the Python programming language, and practiced my new knowledge partially on GitHub via Wikiversity.

I got enough done today, and didn't need to stay up extra late. I still stayed up pretty late though. I hope to make more progress tomorrow.

I skipped marine biology work today, and wrapped up earlier. I plan on going to bed earlier. I hope to try and recover my main laptop tomorrow, and get it functional (as in all data is properly setup) again. I got a pretty good amount of work done today.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

Today was a very good day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 20th 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `05` _(as of May 20th 2021 at 00:12:00 am to 11:59:59 pm)_

[See my year in review for Year 1 (2020-2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021)

I had an OK with development today, and a pretty good day today. It would have been better if I stayed productive.

I focused mainly on documentation work today, and briefly did some HTML scripting. Today, one of my big focuses was on recovering data from the external hard drive that was cloned off my main laptop. This has been an ongoing issue since April 19th 2021, and significant progress on recovery was made again today. I have now managed to locate all the files I thought were lost, and I will now backup my devices before doing any hardware technical support from now on. I am not 100% fine from this though, as the files doubled in size. I don't know why, but about 960 Gigabytes of data is now equivalent to about 1.8 terabytes of data. I am wondering if the HFS+ file system has something to do with it, or if it was something related to the backup. I assume it might be due to encoding (likely not an issue with newline encoding with Macintosh (CR) as it is the same in size as UNIX (LF) and still smaller than Windows (CR)(LF) (cheat sheet: `CR` = `Carriage Return` | `LF` = `Line Feed`) I just need to find out what) and smaller  I am still elated to have all of my data back and unharmed. I plan on transitioning back to the laptop within the next week.

I created 10 more organizations today. I got closer to my organization goal, but I am still not halfway done. I have been struggling to write a status file for all the organizations, I made very minor progress on this today due to distractions and focus issues. 

I got work done today, and had to stay up late to finalize.

I skipped marine biology work again today, and wrapped up earlier. I plan on going to bed earlier. I got a pretty decent amount of work done today. I am hoping it isn't harder to do so tomorrow.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

Today was a very good day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 21st 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `04` _(as of May 21st 2021 at 00:12:00 am to 11:59:59 pm)_

[See my year in review for Year 1 (2020-2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021)

I had a very good day with development today, and a very good day overall.

I focused mainly on documentation work today, and briefly did some project planning. I made massive progress in GitHub organization documentation work, and other documentation work. I am considering another renewal into Degoogle work, I found a good resource (`The social Dilemma` which ironically is on Netflix, and Netflix is not innocent from the subject of the documentary) I am considering making some Wikiversity-inspired edits to my Degoogle articles, and add this as a `multimedia` source. It does a pretty good job at making an overview of the problem with our cyber society, that needs to become a cyberpunk society) I am still going on with this, and I am considering going against more evil, but for now, I feel like I should stick with DeGoogling, as I have the most knowledge regarding this subject, and a "war" against Amazon and Facebook doesn't seem necessary yet to me. Personally, I was able to De-facebook within a week, over 3 years before I began DeGoogling. I never got into Facebook luckily.

I created 10 more organizations today. I got closer to my organization goal, but I am still not halfway done. I am close to halfway now. I have been struggling to write a status file for all the organizations, I made significant progress on this today, but didn't reach the halfway caught-up point. I was laser-focused today, and was able to do lots of work, despite 4 system crashes. Due to this, I plan on doing a hard drive backup tomorrow and going back to my original laptop.

I got work done today, and had to stay up late to finalize.

I skipped marine biology work again today, and wrapped up earlier. I plan on going to bed earlier. I got a pretty good amount of work done today. I am hoping it isn't harder to do so again tomorrow. I am now 3 days behind with Marine Biology work.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

Today was a very good day for development



***

**Comment by seanpm2001** [comment]
***

### Status update: May 22nd 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `03` _(as of May 22nd 2021 at 00:12:00 am to 11:59:59 pm)_

[See my year in review for Year 1 (2020-2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021)

I had an OK day with development today, and a very good day overall.

I focused mainly on documentation work today, and didn't get far with it. I focused heavily on getting my laptop setup again, and I transitioned over from my clone laptop to my original laptop. I have a summary for this whole issue (April 19th 2021 to May 22nd 2021 (1 Month, 3 days))

I have undergone a difficult period in which I was stripped of access to several of my computing resources (most importantly: music, some pictures, and some documents) it started on April 19th 2021 when a Dell repair went wrong and my operating system was unmounted. It is a good thing I had been setting up a backup clone of my laptop since March 31st 2021.

I have gone through numerous efforts to repair and recover the data, and I was able to do so successfully with the help of [Price Computers LLC](http://www.pricecomputersllc.com/) who helped me recover all the data and reinstall the operating system. Huge kudos to them. It would be impossible for me to go on without this fix.

I have recently began to worry about the safety of the data without plugging the hard drive in, and I eventually got the guts to do so, at first finding the data to be possibly corrupt, but later finding out it was all there (the recovered data was placed on a 2 terabyte Western Digital hard drive, formatted with the HFS+ file system. It is a 2 terabyte drive, because the recovery software used duplicated my data, and caused ~900 gigabytes to turn into nearly 1.8 terabytes) further experimentation occurred, and today (Saturday, May 22nd 2021) I have switched back over to my original laptop. I have gotten access back to my music collection, and it has remained stable for multiple days. I also am dealing with software bugs that I know how to fix, and they are occurring less often.

The lesson from this is to never hesitate to backup your laptop before doing a hardware repair, no matter how basic the repair is and how much you trust the person repairing, as things can always go wrong (I was told my laptop lacked a CMOS battery, which supposedly corrupted a part of the BIOS/bootloader when the keyboard was replaced)

This is just a copy-paste of a status from my audio notes, but it is a needed explanation. To signify that I am back on the original laptop, I have put the dock back on the bottom (opposed to the default left side of the screen) and have written notes regarding the laptop.

I created 10 more organizations today. I got closer to my organization goal, but I am still not halfway done. I am close to halfway now. I have been struggling to write a status file for all the organizations, I made significant progress on this today, but didn't reach the halfway caught-up point.

I got work done today, and had to stay up late to finalize. I still ended up failing on my goal of documenting organizations, meaning I will have to do double the work on this tomorrow.

I got caught up on my marine biology work and my DuckDuckGo image work today. I was 4 days behind.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

Today was an OK day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 23rd 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `02` _(as of May 23rd 2021 at 00:12:00 am to 11:59:59 pm)_

[See my year in review for Year 1 (2020-2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021)

I had an very day with development today, and a good day overall.

I focused mainly on documentation work today, and got very far with it. I am now past halfway done with my Organization documentation process, with 103 to go (plus the 10 per day) It took a lot of time today.
 
I created 10 more organizations today. I got closer to my organization goal, but I am still not halfway done.

I got work done today, and had to stay up late to finalize. I am still anticipating my 1 year GitHub anniversary, and my finalization of this organization creation process.

Tonight, I had to install a snap on my computer (youtube-dl) as the apt version was over 1 year and 2 months out of date, which likely was the reason it wasn't working on the past laptop

Today, I also learned more about Ubuntu, and am now less ashamed of using it, although I still dislike the snapcraft system, and the nearly fully-flat interface, but it is still many kilometers better than Windows 10. I am glad that the desktop environment Ubuntu uses (GNOME) maintains some gradient, rather than a depressing and boring flat look like that of Windows 8 and up, Android 5.0 and up, iOS 7 and up, MacOS 10.10 and up, etc. I still await the day that Skeuomorphism returns to relevance.

One cool fact I learned about Ubuntu today was that NASA used Ubuntu as their platform of choice to render the first image of a black hole. I didn't read the whole 6 points Canonical gave, if you are struggling to find a good distribution, that Ubuntu is a good option for new users, and possibly all users. Don't like the interf

I was out of it when I wrote this, sorry. I got a little intoxicated tonight, and I am trying to wrap this work up for the night and go to bed.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

Today was an OK day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 24th 2021

🎂 **Days until 1 year GitHub :octocat: anniversary:** `01` _(as of May 24th 2021 at 00:12:00 am to 11:59:59 pm)_

[See my year in review for Year 1 (2020-2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021)

I had an decent day with development today, and a mixed day overall.

I focused mainly on documentation work today, and got very far with it. I am now further past halfway done with my Organization documentation process, with 84 to go (plus the 10 per day) It took a lot of time today.
 
I created 10 more organizations today. I got closer to my organization goal, but I am still not halfway done.

I got work done today, and had to stay up late to finalize. I am still anticipating my 1 year GitHub anniversary tomorrow, and my finalization of this organization creation process.

I began downloading light-DRM (low level DRM in sites such as YouTube) locked videos on my laptop again, a functionality I have been missing for over a month. My life hasn't been very interesting lately, and I got really depressed today, so it was a bit difficult for me.

I really didn't do very much today outside of documentation. I tried to make a lot of progress on documentating organizations today, which I did, but still at a slow rate. I hope to not repeat todays patterns tomorrow.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

Today was an decent day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 25th 2021



**🎂 1 Year :octocat: GitHub anniversary (May 25th 2020-May 25th 2021)**

[See my year in review for Year 1 (2020-2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021)

I had an very day with development today, and a very good day overall. Today was my 1 year anniversary day on GitHub. It wasn't as exciting as it should have been, but it was still a memorable day today for me.

I focused mainly on documentation work today, and got very far with it. I am now far past halfway done with my Organization documentation process, with 61 to go (plus the 10 per day) It took a lot of time today. I am now going to force myself to work in this project a lot less, limiting myself to 16-17 parts per day, instead of todays 35+

I did some actual programming work today regarding WacOS. It wasn't too signicant of a set of contributions in quality, but it served as a stepping stone for me learning the C programming language a bit better, and a better introduction for header files, although a lot of it was a boilerplate and some of it was pseudocode. I started establishing the 5 file systems in WacOS today, the 5 default file systems the user can choose to install are `EXT4` `OpenZFS` `OpenAPFS` `HFS` and `HFS+`
 
I created 10 more organizations today. I got closer to my organization goal, but I am still not halfway done.

I got work done today, and had to stay up late to finalize. I am still anticipating the finalization of this organization creation process.

I got a lot of socializing done outside today, I am in the process of making a new friend. I have come to note that my humorous smirk is a very valuable part of me, as I have now noted over 4 people can't seem to get enough of it when they see me. More importantly, my new friend has a lot of personality, and we share a lot of interests and beliefs.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

I managed to squeeze Marine Biology and DuckDuckGo documentation in today, I almost didn't have time. I wrote this at 11:22 pm, and I had to be 100% done by 11:59:59:9999

Today was an very good day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 26th 2021



I am still preparing the YIR (Year In Review) but I need to keep gathering more snippet data first, as GitHub still recognizes May 26th 2021 to be in the wrong spot (I don't know how to better describe this)

[See my year in review for Year 1 (2020-2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021)

I had an very day with development today, and a very good day overall. Today was a far less exciting, but I got a lot done.
 
I focused mainly on documentation work today, and got very far with it. I am now far past 77% done with my Organization documentation process, with 40 to go (plus the 10 per day) It took a lot of time today. I am now going to force myself to work in this project a lot less, limiting myself to 16-17 parts per day. I decided to do more today, so I would be less tempted tomorrow, have a more even number, and better prepare myself for my Thursday schedule.

I did some documentation work on WacOS today, giving documentation for the 5 default file systems the user can choose to install are `EXT4` `OpenZFS` `OpenAPFS` `HFS` and `HFS+` however, I am now considering a 6th default system, `BTRFS` as based on EXT4 research, it seems to be a better fit for many people over Ext4. 
 
I created 10 more organizations today. I got closer to my organization goal, but I am still not halfway done.

I got work done today, and had to stay up late to finalize. I am still anticipating the finalization of this organization creation process.

I got a lot of socializing done outside today, I am in the process of making a new friend. It wasn't as interactive today, but it was still a nice visit. I am the family privacy associate when it comes to the technical field with limited power. For this reason, despite this being a high profile visitor, I will not be mentioning them by name for at least 1 year.

Today, I saw a DuckDuckGo ad on TV. I believe this may be the start of a huge upscale to the privacy movement, and hopefully people will begin to realize that their privacy is important. I feel it really sticks it to the evil corporation that came before the ad.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I have to go to bed early tonight, I have a lot planned for tomorrow. I have to see a Nutritionist due to concerns about my vegetarian diet (I will probably just get a supplement, nothing else is going to change) along with my usual extended Thursday work, and possibly also go to a birthday party, and do journaling and other work, a pretty busy day. I had to finalize earlier tonight.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

I can't get marine biology or DuckDuckGo work done tonight, I am too tired.

Today was an very good day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 27th 2021

🎂 **Days until 2 year GitHub :octocat: anniversary:** `363` _(as of May 27th 2021 at 00:12:00 am to 11:59:59 pm)_ 

I am still preparing the YIR (Year In Review) for year 1, but I need to keep gathering more snippet data first, as GitHub still recognizes May 25th 2021 to be in the wrong spot (I don't know how to better describe this)

[See my year in review for Year 1 (May 25th 2020 to May 25th 2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021)

I had a decent day with development today, and a bad day personally.
 
I focused mainly on documentation work today, and got pretty far with it. I am now far past 80% done with my Organization documentation process, with 35 to go (plus the 10 per day) It took a lot of time today. I am now going to force myself to work in this project a lot less, limiting myself to 16-17 parts per day. I did less today, as I didn't have as much time. With Git-image work, I got less than 100 pictures for all 7 games today.

I have been rapidly losing followers lately, it is just an artificial number. I got so used to things that stimulate the mind like commit counts (I need the boost sometimes) but I seem to commonly forget that these don't matter very much in the long run, as quality of work counts better than quantity, and if quality can be done in quantity, then that should be done. I only mention this, as during the process of just creating 3 organizations today, I lost 3 followers. It seemed like I was losing a follower for each organization I created. Not very many people stick around long enough to ask me a question about my projects, so it doesn't matter too much. I have gone from 156 followers to 151 followers in the past few days, still a considerable amount, but mostly an artificial number.

I have noticed that someone is interacting with some of my projects.

I created 10 more organizations today. I got closer to my organization goal, but I am still not halfway done.

I got work done today, and had to stay up late to finalize. I am still anticipating the finalization of this organization creation process.

I had a difficult day today emotionally and productively, due to severe schedule shifts and illogical debates. In general, I shouldn't be looking at today as negatively, but the illogical debate really got to me, and broke down my mood. I can't mention what the debate was about until May 14th 2040 or a different date due to privacy and security concerns.

I have further been considering a new phone, as part of my plan on phasing out Android. I am down to 2 options at the moment, a PinePhone or a Librem 5. It is still a lengthy process, for one, I can't make a virtual machine of my current phone yet, as I don't have a laptop or desktop computer with over 512 gigabytes of free space that is stable enough or has the capacity for a virtual machine of this size (just installation and a single snapshot will take at least 256 Gigabytes) I am trying to find out first if I can still do SMS messaging without a SIM card, as I have gone through a ridiculous situation with my SIM card in the past 2 months, with the SIM card failing at least 80 times.

I also started writing a book today regarding the concept of preservation that is related to several of my key projects. The book is licensed under the GNU General Public License v3.0 and it is going to be released free of charge, like all of my other works. I am currently preparing the release, version 1 is ready, butI just have so many major projects I have to get to first at the moment (organization work, organization documentation work, daily git-image work, software documentation, journaling, audio documentation, video documentation, project Slim (SLIM I of my audio collection) culinary documentation, file sorting, and more)

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

I can't get marine biology or DuckDuckGo work done tonight, I am too tired.

Today was a decent day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 28th 2021

🎂 **Days until 2 year GitHub :octocat: anniversary:** `362` _(as of May 28th 2021 at 00:12:00 am to 11:59:59 pm)_ 



I am still preparing the YIR (Year In Review) for year 1, but I need to keep gathering more snippet data first, as GitHub still recognizes May 25th 2021 to be in the wrong spot (I don't know how to better describe this)



[See my year in review for Year 1 (May 25th 2020 to May 25th 2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021)

I had a decent day with development today, and a better day personally, but I dealt with some depression today.
 
I focused mainly on documentation work today, and got pretty far with it. I am now far past 90% done with my Organization documentation process, with 35 to go (plus the 10 per day) It took a lot of time today. I am now going to force myself to work in this project a lot less, limiting myself to 16-17 parts per day. I did more today, and am down to 21-23 entries before being caught up.

I created 10 more organizations today. I got closer to my organization goal, but I am still not halfway done.

I got work done today, and had to stay up late to finalize. I am still anticipating the finalization of this organization creation process.

I have made a positive observation today regarding my switch to Linux, even with GNOME (which isn't my preferred Linux desktop environment, KDE is (that is until Skeuowie catches on)) that I have significantly reduced my carbon emission rate simply by switching from Windows 10 to Ubuntu with a heavy desktop environment (GNOME 3) Although not perfect, I have reduced consistent CPU usage from 56-90% to 8-25% simply by switching to Linux. This has an impact on the amount of heat my laptop produces, thus reduces carbon emissions. Now if only I could get rid of human tissue, skeleton/bones, vital organs, and so on, then I would be nearly 100% carbon emission free (kidding of course, I can't do that, obviously)

I also started writing a book yesterday (Thursday, May 27th 2021) regarding the concept of preservation that is related to several of my key projects. The book is licensed under the GNU General Public License v3.0 and it is going to be released free of charge, like all of my other works. I am currently preparing the release, version 1 is ready, butI just have so many major projects I have to get to first at the moment (organization work, organization documentation work, daily git-image work, software documentation, journaling, audio documentation, video documentation, project Slim (SLIM I of my audio collection) culinary documentation, file sorting, and more) I am now freely creating new documents without restriction, which is a big step of progress for me, as I have been struggling on this goal for over a year. Hopefully soon I can start writing down my childhood stories again before I get too old/someone dies.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation.

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate.

I got caught up with the DuckDuckGo and marine biology work today, although I stayed up really late to finish.

Today was a decent day for development

***

**Comment by seanpm2001** [comment]
***

### Status update: May 29th 2021

🎂 **Days until 2 year GitHub :octocat: anniversary:** `361` _(as of May 29th 2021 at 00:12:00 am to 11:59:59 pm)_ 



I am still preparing the YIR (Year In Review) for year 1, but I need to keep gathering more snippet data first, as GitHub still recognizes May 25th 2021 to be in the wrong spot (I don't know how to better describe this) 



[See my year in review for Year 1 (May 25th 2020 to May 25th 2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021) 

I had a decent day with development today, and a much better day personally.
 
I focused mainly on documentation work today, and got pretty far with it. I am now far past 99.01% done with my Organization documentation process, with 3 to go (plus the 10 per day) It took a lot of time today. I am all caught up, except for 3 entries I am having difficulty finding.

I created 10 more organizations today. I got closer to my organization goal, and I am now currently past halfway done, but not too close to being done. It is still expected to take up to June 21st 2021 or longer. 

I got work done today, and had to stay up late to finalize. I am still anticipating the finalization of this organization creation process. I made significant progress catching up on writing entries today, but now I have found that I can't list 3 organizations, not 2, and not 0. It is going to take some searching to find them.

I also started writing a book recently (on Thursday, May 27th 2021) regarding the concept of preservation that is related to several of my key projects. The book is licensed under the GNU General Public License v3.0 and it is going to be released free of charge, like all of my other works. I am currently preparing the release, version 1 is ready, butI just have so many major projects I have to get to first at the moment (organization work, organization documentation work, daily git-image work, software documentation, journaling, audio documentation, video documentation, project Slim (SLIM I of my audio collection) culinary documentation, file sorting, and more) I am now freely creating new documents without restriction, which is a big step of progress for me, as I have been struggling on this goal for over a year. Hopefully soon I can start writing down my childhood stories again before I get too old/someone dies. 

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation. 

I didn't get any work done on the year in review project today. This project I feel isn't going to be developed at the current rate. 

I got caught up with the DuckDuckGo and marine biology work today, although I stayed up really late to finish. 

Today was a decent day for development. 

***

**Comment by seanpm2001** [comment]
***

### Status update: May 30th 2021

🎂 **Days until 2 year GitHub :octocat: anniversary:** `360` _(as of May 30th 2021 at 00:12:00 am to 11:59:59 pm)_ 



I am still preparing the YIR (Year In Review) for year 1, but I need to keep gathering more snippet data first, as GitHub still recognizes May 25th 2021 to be in the wrong spot (I don't know how to better describe this) 



[See my year in review for Year 1 (May 25th 2020 to May 25th 2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021) 

I had a decent day with development today, and a stressful difficult day personally.

I focused mainly on documentation work today, and got pretty far with it. I got to work a lot later today, and stayed caught up with GitHub organization documentation, minus the 3 missing entries. I raised 2 issues today, 1 regarding the 3 missing directories, and 1 for future plans for the project.

I created 10 more organizations today. I got closer to my organization goal, and I am now currently past 52% done, but not too close to being done. It is still expected to take up to June 21st 2021 or longer. 

I got work done today, and had to stay up late to finalize. I am still anticipating the finalization of this organization creation process.

I also started writing a book recently (on Thursday, May 27th 2021) regarding the concept of preservation that is related to several of my key projects. The book is licensed under the GNU General Public License v3.0 and it is going to be released free of charge, like all of my other works. I am currently preparing the release, version 1 is ready, butI just have so many major projects I have to get to first at the moment (organization work, organization documentation work, daily git-image work, software documentation, journaling, audio documentation, video documentation, project Slim (SLIM I of my audio collection) culinary documentation, file sorting, and more) I am now freely creating new documents without restriction, which is a big step of progress for me, as I have been struggling on this goal for over a year. Hopefully soon I can start writing down my childhood stories again before I get too old/someone dies. 

I did many forks today, which took a lot of time, but now an additional mass-language project is up to date with its categorization, so this shouldn't have to be done again for a while. I surpassed 7.1K stars today (projects I am starring) I am currently considering some new project ideas for when I have more time.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation. 

I got to have some dedicated laptop time to watch videos today, something I haven't done in over a month. Things are slowly returning back to normal for me.

I got some work done on the Year in Review project, GitHub is no longer giving me scoped data for the past year before May 31st 2021, so now I have all the images ready. This project I feel isn't going to be developed at the current rate. 

I got caught up with the DuckDuckGo and marine biology work today, although I stayed up really late to finish. 

Today was a decent day for development. 

***

**Comment by seanpm2001** [comment]
***

### Status update: May 31st 2021

🎂 **Days until 2 year GitHub :octocat: anniversary:** `359` _(as of May 31st 2021 at 00:12:00 am to 11:59:59 pm)_ 

:shipit: **Days until GitHub :octocat: organization :atom: daily process of late Spring 2021 :electron: is complete:** `~21` (as of May 31st 2021 at 00:12:00 am to 11:59:59 pm)_ 



I am still preparing the YIR (Year In Review) for year 1, but I need to keep gathering more snippet data first, as GitHub still recognizes May 25th 2021 to be in the wrong spot (I don't know how to better describe this) 



[See my year in review for Year 1 (May 25th 2020 to May 25th 2021)](https://github.com/seanpm2001/seanpm2001/blob/master/Special/Year-in-Review/2020-2021) 

I had a decent day with development today, and a good day personally. I have been low on time lately, and I have finally pinpointed it to my GitHub organization work and a lack of multi-tasking, something I should have noticed a lot earlier. I will work harder on spending time with my family while doing this work.

I focused mainly on documentation work today, and got pretty far with it. I got to work a lot later today, and stayed caught up with GitHub organization documentation, minus the 3 missing entries.

I created 10 more organizations today. I got closer to my organization goal, and I am now currently past 53.5% done, but not too close to being done. It is still expected to take up to June 21st 2021 or longer. 

I got work done today, and had to stay up late to finalize. I am still anticipating the finalization of this organization creation process.

 

I did many forks today, which took a lot of time, but now an additional mass-language project is up to date with its categorization, so this shouldn't have to be done again for a while. I surpassed 7.2K stars today (projects I am starring) I am currently considering some new project ideas for when I have more time.

I began learning about the ZenScript language today, a bit obscure with little documentation online, but still popular enough for 1 project to be recommended to me written in this language, and for GitHub to recognize it as a programming language. I am looking into it.

Along with all of this, I did the usual Git-image upload work, and standard end-of-day documentation. 

I got to have some dedicated laptop time to watch videos today, something I haven't done in over a month. Things are slowly returning back to normal for me. I got more time for this today.

I start with a clean monthly slate tomorrow for the month of June, but I have been dreading the possible consequences of my monthly Tuesday gaming sessions for the past 8 days.

Lately, I have been customizing Firefox a lot more. It has been enjoyable. I also learned about all the customizations I can do to the system monitor today. I made some changes, one was a bit far (enabling all disks to be viewed) and one was easier for me to process (making the CPU graph a 3D graph, rather than 8 different constantly intersecting lines. It also looks a lot cooler.

Today, I did some actual programming work, working on a new backup tool concept known as an app backpack, short for app/program-data backup pack. I also briefly worked on the PrestigePreview software library today, and did some work on getting my audio collection back up to date. I am not completely done with getting my audio collection Git framework back up to date.

I got some work done on the Year in Review project, GitHub is no longer giving me scoped data for the past year before May 31st 2021, so now I have all the images ready. This project I feel isn't going to be developed at the current rate. 

I managed to have the time to do DuckDuckGo and Marine Biology work, but it was difficult to fit in. 

Today was a decent day for development. 

***
